packages/core/admin/admin/src/hooks/useAdminUsers/useAdminUsers.js

```diff
@@ -0,0 +1,51 @@
+import { useQuery, useQueryClient } from 'react-query';
+import { useFetchClient } from '@strapi/helper-plugin';
+import { stringify } from 'qs';
+
+/**
+ * @param {int} [undefined] userId - a User id
+ * @param {object} queryParams - a query params object
+ * @param {object} queryOptions - an object extending the default react-query options
+ */
+
+// eslint-disable-next-line default-param-last
+export function useAdminUsers(userId = '', queryParams = {}, queryOptions = {}) {
+  const QUERY_KEY = ['users', userId, queryParams];
+  const queryString = stringify(queryParams, { encode: false });
+
+  const queryClient = useQueryClient();
+  const { get } = useFetchClient();
+
+  const refetchQueries = async () => {
+    await queryClient.refetchQueries(QUERY_KEY);
+  };
```

Comment:
doesn't useQuery return a `refetch` function?

Reply:
I'll check that 👍🏼